src/main/java/com/ably/kafka/connect/config/ConfigValueEvaluator.java

```diff
@@ -6,7 +6,26 @@
 import java.nio.charset.StandardCharsets;
 
 public class ConfigValueEvaluator {
+    /*
+    Result class wrapping the evaluated value and whether it should be skipped or not
+    This has been created to wrap the result of the evaluation of a config value when skippability is involved.
+     * */
+    public static class Result{
+        boolean skip;
```

Comment:
Could this field, and `value` below it, be `private`?

Reply:
Of course. I also made them `final` as there was no reason for them to be mutable e99f2580953029950c3c37284d7e1aed97d926a4